source/people/mummah.rst

```diff
@@ -0,0 +1,56 @@
+Katie Mummah
+===================
+
+.. image::  mummah.jpg
+    :align: center
+    :width: 300
+
+Contact Information
+--------------------
+
+| 440 Engineering Research Building
+| 1500 Engineering Drive
+| Madison, WI 53706
+
+| `mummah@wisc.edu <mailto:mummah@wisc.edu>`_
+| `katiemummah@gmail.com <mailto:katiemummah@gmail.com>`_
+
+Find me on:
+|gh_icon|_
+|li_icon|_
+|tw_icon|_
+|wb_icon|_
+
+Graduate Appointment
+---------------------
+
+Fellow, Consortium for Verification Technology
+
+Research Interests
+-------------------
+
+ * Fuel cycle modeling
+ * Nuclear nonproliferation
+
+Education
+----------
+
+ * University of Wisconsin, Madison, WI [Graduate Student, Nuclear Engineering & Engineering Physics]
+ * University of Illinois, Urbana, IL [BS Nuclear, Plasma, and Radiological Engineering, 2017]
+
+
+.. |gh_icon| image:: github.png
+              :width: 32
+.. _gh_icon: https://github.com/nuclearkatie
+
+.. |tw_icon| image:: twitter.png
+              :width: 32
+.. _tw_icon: https://twitter.com/nuclearkatie
+
+.. |li_icon| image:: linkedin.png
+              :width: 32
+.. _li_icon: https://www.linkedin.com/in/nuclearkatie
+
+.. |wb_icon| image:: website-logo.png
+              :width: 32
+.. _wb_icon: https://nuclearkatie.com
```

Comment:
This link was failing for me, but seemed to work with just `http` (rather than `https`)